docs/src/content/docs/id/getting-started.mdx

```diff
@@ -5,64 +5,110 @@ description: Pelajari cara membuat website dokumentasi baru Anda dengan Starligh
 ```
 
 </TabItem>
 </Tabs>
 
-Perintah tersebut akan membuat [direktori proyek](/id/guides/project-structure/) baru berisikan seluruh file dan konfigurasi untuk website Anda.
+Perintah tersebut akan membuat [direktori proyek](/id/guides/project-structure/) baru berisikan seluruh file dan konfigurasi yang diperlukan untuk situs Anda.
 
 :::tip[Lihat contohnya]
-Cobalah Starlight di browser:
+Cobalah Starlight di browser Anda:
 [buka dengan StackBlitz](https://stackblitz.com/github/withastro/starlight/tree/main/examples/basics).
 :::
 
-## Membuat konten dengan Starlight
+### Mulai server pengembangan
 
-Starlight sudah siap untuk ditambahkan konten baru, atau pindahkan konten lama Anda!
+Ketika bekerja secara lokal, [server pengembangan Astro](https://docs.astro.build/en/reference/cli-reference/#astro-dev) memungkinkan Anda melihat pratinjau pekerjaan Anda dan secara otomatis menyegarkan peramban ketika Anda membuat perubahan.
 
-### Format file
+Di dalam direktori proyek Anda, jalankan perintah berikut untuk memulai server pengembangan:
 
-Starlight mendukung pembuatan konten dalam bentuk Markdown and MDX. (Anda dapat menambahkan support untuk Markdoc dengan menginstall integrasi eksperimental [Astro Markdoc](https://docs.astro.build/en/guides/integrations-guide/markdoc/).)
+<Tabs>
+<TabItem label="npm">
 
-### Tambah halaman baru
+```sh
+npm run dev
+```
 
-Tambah halaman baru secara otomatis dengan cara membuat file `.md` or `.mdx` di `src/content/docs/`. Tambahkan folder untuk mengatur konten Anda, dan membuat beberapa segmen _path_ bertingkat:
+</TabItem>
+<TabItem label="pnpm">
 
+```sh
+pnpm dev
 ```
-src/content/docs/hello-world.md => your-site.com/hello-world
-src/content/docs/guides/faq.md => your-site.com/guides/faq
+
+</TabItem>
+<TabItem label="Yarn">
+
+```sh
+yarn dev
 ```
 
-### Frontmatter yang type-safe
+</TabItem>
+</Tabs>
+
+Ini akan mencatat pesan ke terminal Anda dengan URL pratinjau lokal Anda.
+Buka URL ini untuk mulai menjelajahi situs Anda.
+
+### Tambahkan konten
+
+Starlight siap untuk ditambahkan konten baru, atau pindahkan konten Anda yang sudah ada!
+
+#### Format file
+
+Starlight mendukung penulisan konten dalam bentuk Markdown dan MDX tanpa memerlukan konfigurasi.
+Anda dapat menambahkan dukungan untuk Markdoc dengan menginstall [integrasi Astro Markdoc](https://docs.astro.build/en/guides/integrations-guide/markdoc/) eksperimental.
+
+#### Tambahkan halaman
+
+Tambahkan halaman baru ke situs Anda dengan cara membuat file `.md` atau `.mdx` di `src/content/docs/`.
+Gunakan sub-folder untuk mengatur file Anda dan untuk membuat beberapa segmen _path_.
+
+Sebagai contoh, struktur file berikut ini akan menghasilkan halaman di `example.com/hello-world` dan `example.com/guides/faq`:
+
+import FileTree from '../../../components/file-tree.astro';
+
+<FileTree>
+
+- src/
+  - content/
+    - docs/
+      - guides/
+        - faq.md
+      - hello-world.md
+
+</FileTree>
+
+#### Frontmatter yang type-safe
```

Comment:
The Markdown file example right below usually gets the title and description translated for other languages, would you mind translating that too?